src/Donations/Endpoints/DonationUpdateAttributes/PaymentInformation/FeeAmountRecoveredAttribute.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Give\Donations\Endpoints\DonationUpdateAttributes\PaymentInformation;
+
+use Give\Donations\Endpoints\DonationUpdateAttributes\DonationUpdateAttribute;
+use Give\Donations\Models\Donation;
+use Give\Framework\Support\ValueObjects\Money;
+
+/**
+ * Class FeeAmountRecoveredAttribute
+ *
+ * @unreleased
+ */
+class FeeAmountRecoveredAttribute extends DonationUpdateAttribute
+{
+    /**
+     * @inheritDoc
+     */
+    public static function getId(): string
+    {
+        return 'feeAmountRecovered';
+    }
+
+    /**
+     * @inheritDoc
+     */
+    public static function getDefinition(): array
+    {
+        return [
+            'type' => 'number',
+            'required' => false,
+            'minimum' => 0,
+            'sanitize_callback' => function ($param) {
+                return floatval($param);
+            },
```

Comment:
We want to avoid turning currency amounts into floats whenever possible. Keep it as a string.

Reply:
Resolved e3c8e6c82f9bc319df0631ea94b5caa65490da43